config/secrets.yml.dev

```diff
@@ -91,10 +91,10 @@ dev_settings: &DEV_SETTINGS
     adapter: :net_http_persistent
   newrelic:
     <<: *NEWRELIC_SECRETS
-    app_name: USA Search (PUT YOUR NAME HERE)
+    app_name: USA Search (PUT YOUR NAME HERE) (Development)
```

Comment:
As long as we're updating this, let's keep it consistent with the cookbooks*:
```
app_name: search.gov (PUT YOUR NAME HERE) (<%= Rails.env %>)
```
We still need to interpolate the environment, as these secrets are used for both test and dev environments.

("USA Search" was our old name, and the old repo name was "usasearch". Those terms are still around, but we're updating them to "search.gov" when possible.)

Reply:
Ah, I didn't realize these were used in the test environment as well. PR has been updated.